sys/auto_init/auto_init.c

```diff
@@ -24,6 +24,10 @@
 #include "bmp180.h"
 #endif
 
+#ifdef MODULE_BME280
```

Comment:
Have a look at this document: https://github.com/RIOT-OS/RIOT/wiki/Guide:-Writing-a-device-driver-in-RIOT